content/Maximo/VISUAL-INSPECTION-ACTION.md

```diff
@@ -0,0 +1,214 @@
+![](_attatchments/mvi-action/MVI-action-detection-Step4.3.1.png)
+
+4. Once it is done training, you will see the details regarding accuracy, precision and other characteristics about the model.
+
+5. You can also click the **Hamburger** icon (a stacked bar) on the top left to view the list of page navigation options. Select **Models** from the list to view your trained model in the list of trained models. The click on the **Water Bottle Action Detection Model** you just trained to see more details
+
+![](_attatchments/mvi-action/MVI-action-detection-Step4.5.png)
+
+*Note: If you are in a live demo setting or there are not enought GPUs open in the environment to train the dataset and create a model, [**here**](_attatchments/mvi-action/waterbottle-action-pretrained-model.zip) is the action detection model shown in this tutorial that you can download and upload to use. Once downloaded, go to the **Models** section and click on the **Upload files** hyperlink and select the file you just downloaded.*
+
+### Step 5: Model Testing
+
+**Actions:**
+
+Once model training is complete and at a desired accuracy (if you made your own dataset and your accuracy is lower than desired, try adding more videos to your dataset with as little variation as you can), the model should be tested before making it generally available. In the validation phase, you will pass additional water bottle videos in action to the model that were not part of the training dataset (called validation or test images) and let the model auto-detect the action being performed in the video. This will ensure that the model we have trained is able to understand and detect the correct action being performed in the video. 
+
+Using a dataset that was not part of the model-building phase is a standard process to check the model performance (accuracy). This helps assure that the patterns detected during model training are generalizable (also exist in the never-before-seen videos) and can be used in the real world to predict new “unseen” videos. 
+
+1. To test your model, once in the details of your trained model, click on the blue **Deploy model** button in the top right corner. *Reminder: Make sure to add your initials again to the deployed model name.* 
+
+![](_attatchments/mvi-action/MVI-action-detection-Step5.1.1.png)
+
+2. Click again on the **Hamburger** icon in the top left, and then on the **Deployed models** section. On the **Deployed models** screen, under the Name column, click on the blue hyperlinked **Water bottle action detection dataset**.
+
+![](_attatchments/mvi-action/MVI-action-detection-Step5.2.png)
+
+3. In this section, you will see some of the details and information about the deployed model and the API endpoint for any potential future app integration. Upload your new test videos in the box outlined in red in the below screenshot. 
+
+![](_attatchments/mvi-action/MVI-action-detection-Step5.3.png)
+
+4. First, upload a video of you twisting the cap on the water bottle. 
+
```

Comment:
Again, maybe add a section to talk about downloading, unzipping, and utilizing the pre-recorded untrained videos if they prefer.